File: rohtext.py
#!/usr/bin/python

from sys import argv
import regex

with open(argv[1]) as datei:
     text = datei.read()
     rohtext = regex.search(r'\\\opening\{ .*, \}(.*)\\\closing\{ .* \}', text, regex.S|regex.M)
     rohtext = rohtext.group(1)
     ergebnis = regex.sub('(?:\\\.*\{|\}|\\\\-|\\\)', '', rohtext) 
     print(ergebnis)
             

File: rohtext.pl
#!/usr/bin/perl

use Modern::Perl;
use Data::Dumper;

$/ = undef;
open my $latex, "<", $ARGV[0];

my $text = <$latex>;

$text =~ m/\%\%ANFANG(.*)\%\%ENDE/gismo;

my $rumpf = $1;

$rumpf =~ s/^\s//;

print $rumpf;
